.github/workflows/release.yaml

```diff
@@ -0,0 +1,123 @@
+name: release
+on:
+  pull_request:
+    paths:
+    - '.github/workflows/release.yaml'
+    - 'dist/*'
+    - 'tools/*'
+  push:
+    branches:
+    - main
```

Comment:
We can remove that for the time being, but my feeling is that we'd want to publish these sort of "nightly" snapshot packages somewhere, maybe to GitHub Packages or something. They aren't really nightly either, since they only get built on a merge to main, so I guess `-snapshot` is a better convention for those, if we decide to produce them.

Reply:
I've modified this a bit so that we still run the workflow on pushes to main, but no release is created and no packages are uploaded anywhere (except as artifacts associated to the workflow run, which might be handy). We can remove this entirely if it's not useful, or if it is expensive to run. I'll leave it up to you all on whether or not to keep it in.

Reply:
👍